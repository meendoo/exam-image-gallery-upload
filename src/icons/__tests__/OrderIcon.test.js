import React from "react";
import OrderIcon from "../OrderIcon";
import renderer from "react-test-renderer";

describe("OrderIcon", () => {
  it("should render as it is", () => {
    const orderIcon = renderer.create(<OrderIcon />);
    expect(orderIcon).toMatchSnapshot();
  });
  it("should render properly if props are given", () => {
    const orderIconWithProps = renderer.create(<OrderIcon width={200} height={200} fill="white" />);
    expect(orderIconWithProps).toMatchSnapshot();
  });
});
