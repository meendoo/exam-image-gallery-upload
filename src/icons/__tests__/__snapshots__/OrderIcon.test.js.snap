// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderIcon should render as it is 1`] = `
<svg
  height="115px"
  viewBox="0 0 115 115"
  width="115px"
  x="0px"
  y="0px"
>
  <g
    fill="none"
    fillRule="evenodd"
    stroke="none"
    strokeWidth="1"
  >
    <g
      fill="currentColor"
      fillRule="nonzero"
    >
      <path
        d="M103.9,71.9L103.9,71.9v12.7l-29.9,30l-29.9-30V71.9l25.4,25.5V55.7h9v41.6L103.9,71.9z M11.1,43.1L11.1,43.1V30.4L41,0.4l29.9,30v12.7L45.5,17.6v41.6h-9V17.6L11.1,43.1z"
      />
    </g>
  </g>
</svg>
`;

exports[`OrderIcon should render properly if props are given 1`] = `
<svg
  height={200}
  viewBox="0 0 115 115"
  width={200}
  x="0px"
  y="0px"
>
  <g
    fill="none"
    fillRule="evenodd"
    stroke="none"
    strokeWidth="1"
  >
    <g
      fill="white"
      fillRule="nonzero"
    >
      <path
        d="M103.9,71.9L103.9,71.9v12.7l-29.9,30l-29.9-30V71.9l25.4,25.5V55.7h9v41.6L103.9,71.9z M11.1,43.1L11.1,43.1V30.4L41,0.4l29.9,30v12.7L45.5,17.6v41.6h-9V17.6L11.1,43.1z"
      />
    </g>
  </g>
</svg>
`;
